util/tracing/visualization/README.md

```diff
@@ -1,25 +1,40 @@
-# Trace sequence diagram visualiser
+# Trace sequence diagram visualizer
 
-This is a 1st iteration of a prototyping tool for constructing a sequence diagram 
-out of the traces.
-It operates over the csv files generated by `trace_to_csv`.
+The `fedsd` utility is a tool that reports the interactions (exchanged messages)
+between federates and the RTI in a sequence-diagram-like format.
+To use `fedsd`, you need to first obtain an execution trace. To do this, enable
+the tracing mechanism in your Lingua Franca program by setting the `tracing`
+target property to `true` and then compile and run the program. During program
+execution, a set of `.lft` files will be generated.
+
+After program execution, `fedsd` is used to compile the tracing information into
+a diagram. The tool first transforms each `.lft` file into a `.csv` file, by
+internally running `trace_to_csv`. It then aggregates the data from all `.csv`
+files and produces a diagram on the basis of it.
+
+# Installing
+
+To install `fedsd`, change directory to `lingua_franca/util/tracing` and run `make install`.
 
 # Running
 
-Once the `.lft` files collected and tranformed into `csv` files, run:
+In case the federation is launched using the `bash` script under `bin`, an `.lft` trace
+file will be generated for each of the federates, in addition to `rti.lft`. This latter
+contains the RTI trace.
+If, however, the federation is launched manually, then the `RTI` command should be passed
+the `-t` flag in order to make sure that it, too, has tracing enabled.
 ```
-$ python3 sd_gen.py -r <rti_trace.csv> -f <fed0_trace.csv> ... <fedn_trace.csv>
+$ RTI -n <number_of_federates> -t
 ```
 
-The output is an html file with the svg in it.
-
-# Current problems
-
-- The collected traces are not complete. They need to be checked for correcteness as well.
-- All arrows are horizontal and can be duplicated. Need further processing to derive the connections 
-for that.
-- The scale needs exploration
+For convenience, the RTI and all federates should be launched from the same directory.
```

Comment:
```suggestion
It is most convenient to launch the RTI and all federates from the same working directory so that they will all write their trace file to that directory.
```